src/ucs/config/parser.c

```diff
@@ -1614,15 +1614,21 @@ void ucs_config_parse_config_file(const char *dir_path, const char *file_name,
         .section_info = {.name = "",
                          .skip = 0}
     };
-    char file_path[MAXPATHLEN];
+    char *file_path;
     int parse_result;
     FILE* file;
+    ucs_status_t status;
+
+    status = ucs_string_alloc_path_buffer(&file_path, "file_path");
+    if (status != UCS_OK) {
+        goto out;
+    }
```

Comment:
Added `ucs_string_alloc_formatted_path` function since we're not always using `ucs_string_alloc_path_buffer` in conjuction with `ucs_snprintf_safe`, for example `ucs_netif_bond_ad_num_ports`